content/docs/command-reference/checkout.md

```diff
@@ -33,15 +34,19 @@ for more details.
 
 The execution of `dvc checkout` does the following:
 
-- Scans the `.dvc` and `dvc.lock` files to compare against the data files or
-  directories in the <abbr>workspace</abbr>. DVC knows which data
-  (<abbr>outputs</abbr>) match because the corresponding hash values are saved
-  in the `outs` fields in those files. Scanning is limited to the given
+- Scans all `dvc.lock` and `.dvc` files to compare the hash values saved in
+  their `outs` fields against the actual data files or directories in the
+  workspace (similar to `dvc status`). Scanning is limited to the given
   `targets` (if any). See also options `--with-deps` and `--recursive` below.
 
-- Missing data files or directories are restored from the <abbr>cache</abbr>.
-  Those that don't match with any DVC-file are removed. See options `--force`
-  and `--relink`. A list of the changes done is printed.
+- Missing data files or directories are restored from the cache. Those that
+  don't match with any DVC-file are removed. See options `--force` and
+  `--relink`. A list of the changes done is printed.
+
+> Note that `dvc checkout` supports granularity for files found in tracked
```

Comment:
Let's just describe somewhere that target can be (list of things)

Reply:
It's already in the **Synopsis**. But please notice my previous https://github.com/iterative/dvc.org/pull/1384#discussion_r453969894 on this matter.

Reply:
Synopsis is probably not enough. Description should probably include in some way some human-readable meaning of the `targets`, `path`, etc.

Reply:
OK I shortened these paragraphs as much as possible then. It's the only place where granularity is mentioned (other than in some examples.

Reply:
The notes are right next to the existing explanations of `targets`, some of which I added or improved.